src-gen/main/java/javaslang/Tuple1.java

```diff
@@ -104,7 +104,7 @@ public T1 _1() {
 
     @Override
     public Seq<?> toSeq() {
-        return List.of(_1);
+        return List.ofAll(_1);
```

Comment:
See [this comment](https://github.com/javaslang/javaslang/issues/721#issuecomment-154803154) regarding readability.